pkg/controller/controller.go

```diff
@@ -77,6 +78,24 @@ func (c AppGwIngressController) Process(event QueuedEvent) error {
 	// Get all the ingresses and services
 	ingressList := c.k8sContext.GetHTTPIngressList()
 	serviceList := c.k8sContext.GetServiceList()
+	managedTargetsList := c.k8sContext.GetAzureIngressManagedTargets()
+	prohibitedTargetsList := c.k8sContext.GetAzureProhibitedTargets()
+
+	{
```

Comment:
did we intentionally created these blocks ?

Reply:
Yes! These blocks are a) temporary - using them for debug and will delete them as soon as this project is done b) used for narrow scoping of the `managedTargets` and `prohibitedTargets` (helping the garbage collector ;)